policy/scan/local_scanner.go

```diff
@@ -374,14 +375,20 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		}
 
 		// attach the asset details to the assets list
+		// also override asset name if flag --asset-name was used
 		for i := range assets {
 			log.Debug().Str("asset", assets[i].asset.Name).Strs("platform-ids", assets[i].asset.PlatformIds).Msg("update asset")
 			platformMrn := assets[i].asset.PlatformIds[0]
 			assets[i].asset.Mrn = platformAssetMapping[platformMrn].AssetMrn
 			assets[i].asset.Url = platformAssetMapping[platformMrn].Url
+			assetName := viper.GetString("asset-name")
```

Comment:
we should pass that in from the caller of this func, we dont want to use viper here

ideally you want to attach this name to the root asset (cliAsset)